tests/unit/test_bot.py

```diff
@@ -67,20 +68,23 @@ async def test_extracts_urls(self, bot: OdesliBot):
         assert spotify.url == (
             'https://open.spotify.com/track/1gfzgfcrmkn2yTWuVGhCgh'
         )
-        youtube_music = urls[9]
+        spotify = urls[9]
+        assert spotify.platform_key == 'spotify'
+        assert spotify.url == ('https://link.tospotify.com/pfc3erwl2ab')
```

Comment:
I think redundant parentheses can be removed.

Reply:
done